.github/actions/clean-up-gradle-cache/action.yml

```diff
@@ -0,0 +1,10 @@
+name: "Clean up gradle cache workflow"
+description: "Remove some files from the Gradle cache, so they aren't cached by GitHub Actions. Restoring these files from a GitHub Actions cache might cause problems for future builds."
```

Comment:
question: I know this was done before, but I have a few questions to the code:
1. Why do you think, this might cause problems for the future builds? If the problems happen, is there any other way of just cleaning the case in the situation?
2. You've had set up caching by using `gradle/gradle-build-action@v2`, why you wont to disable it now?
3. There is an option to disable cache in gradle-build-action plugin: https://github.com/gradle/gradle-build-action#caching, why not use this option?

Reply:
1. We have it because I got it from Wire's CI setup... I think we could remove it for now, see if the problems occur, and eventually restore it. What do you think?
2. I don't want to disable it, but I see it's both in `prepare-android-env` and `run-unit-test`
3. I didn't know about this option.

Reply:
| I think we could remove it for now, see if the problems occur, and eventually restore it. What do you think?
Good! ;)